src/Integration.TeamExplorer.UnitTests/app.config

```diff
@@ -94,6 +94,10 @@
         <assemblyIdentity name="Microsoft.CodeAnalysis.Workspaces.Desktop" publicKeyToken="31bf3856ad364e35" culture="neutral"/>
         <bindingRedirect oldVersion="0.0.0.0-1.0.0.0" newVersion="1.0.0.0"/>
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
Needed for SectionControllerTests.
One of the packages in the meta package is now pulling System.Threading.Tasks.Extensions.4.5.4, in which the assembly version was changed from 4.2.0.0 to 4.2.0.1. This creates a conflict in the moq package which was fixed in later moq versions, but for now this works too.

github.com/moq/moq4/issues/873